### DISCUSSION TITLE
I'm using this algorithm to get non-silent speech, but it's not recognized by gcp.

### DISCUSSION THREAD

**Discussion by cumthxy** [snakers4/silero-vad#399] [Q&A]
I'm using this algorithm to get non-silent speech, but it's not recognized by gcp.

i have a 60 seconds voice. and i use get_speech_timestamps to get nosound  voice .but can't speech to text by gcp.
and the raw 60seconds voice can speech to text by gcp.

it is sampling_rate is 8000.

**Comment by snakers4** [comment]
Hi,

Please send your audio.
Also what is GCP?

**Reply by cumthxy** [reply]
gcp is a goole speech to text service. 
this is the raw wav.  the vad algorithm   get_speech_timestamps can split. after that can't speech to text by gcp.
[raw_wav.zip](https://github.com/snakers4/silero-vad/files/13558188/raw_wav.zip)

**Comment by cumthxy** [comment]
I found many examples of this. but can't found how to solve it.
this also a case . the raw wav can speech to text by gcp or azure service. 
[raw_wav1.zip](https://github.com/snakers4/silero-vad/files/13558238/raw_wav1.zip)

but after i use this:
 `  wav = read_audio(input_file, sampling_rate=8000)
    speech_timestamps = get_speech_timestamps(wav, model, sampling_rate=8000)
    if len(speech_timestamps) > 0:
        save_audio(output_file,
                   collect_chunks(speech_timestamps, wav), sampling_rate=8000)`

the output_file wav can't speech to text by gcp or azure service..
this is the processed wav.
[text_wav.zip](https://github.com/snakers4/silero-vad/files/13558270/text_wav.zip)
